pages/osx/defaults.md

```diff
@@ -0,0 +1,23 @@
+# defaults
+
+> Access OS X user defaults
+
+- read system default value
+
+`defaults read {{domain}} {{key}}`
+
+- read default values of applications
```

Comment:
should it be application, not applicationS, considering the fact that you are talking about some particular {{app_name}}?